### DISCUSSION TITLE
[GSoC-Week6] Prepare JuliaCon lightning talk

### DISCUSSION THREAD

**Discussion by Sid-Bhatia-0** [JuliaReinforcementLearning/GridWorlds.jl#163] [Ideas]
cc @findmyway @jonathan-laurent

**Last week:**

I reimplemented some more environments as per the new design. #159 #160 #161 #162 
My JuliaCon 2021 lightning talk proposal got accepted. I created the first draft for the contents for the talk and received valuable feedback on it.

**This week:**

This week, I will be focusing on preparing and recording the JuliaCon talk. The resources for my talk will be added to https://github.com/Sid-Bhatia-0/JuliaCon2021Talk

Work on revamping the package will be resumed after I submit the recording of the talk. In my opinion, I should be able to polish this package and release v0.5.0 before JuliaCon 2021.

**Comment by findmyway** [comment]
Nice, also note that since `v0.5.0` is a breaking change, related experiments in `RLZoo` also need to be updated.

**Reply by Sid-Bhatia-0** [reply]
Right. Thanks for pointing that out. By the way, do you have any plans to pin the versions of external repositories for RLZoo experiments like GridWorlds.jl? It would be nice to have a system wherein RLZoo tests don't break unexpectedly because of changes in external packages.

**Reply by findmyway** [reply]
The version used in tests is pinned. But the `Requires.jl` is used in `ReinforcementLearningExperiments`, so we can not pin it at least for now.

**Comment by jonathan-laurent** [comment]
Good luck preparing and recording your talk. I am sure you will do a great job!

**Reply by Sid-Bhatia-0** [reply]
Thanks! ❤️
